### ISSUE TITLE
release bundle does not follow standard

### ISSUE THREAD

**Issue by Himself65** [toeverything/blocksuite#337]
In our current release package, there're some issues.

1. We shouldn't release `src` folders.
2. pure ESM module may cause Next.js SSR error
3. `entry` `module` fields of the package.js may incorrect

```bash
Server Error
SyntaxError: Named export 'Utils' not found. The requested module '@blocksuite/store' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@blocksuite/store';
const { Utils: Bl, Text: Rp, BaseBlockModel: Fn, Signal: Wt } = pkg;
```

**Comment by lawvs** [comment]
Since we configured the TypeScript `sourceMap`, releasing `src` folders is very helpful for debugging.

**Comment by doodlewind** [comment]
This should be fixed by #387 and its following PRs.
